cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -676,6 +677,8 @@ filterConfigureFlags flags cabalLibVersion
           -- We add a Cabal>=3.11 constraint before solving when multi-repl is
           -- enabled, so this should never trigger.
           configPromisedDependencies = assert (null $ configPromisedDependencies flags) []
+        , -- Cabal < 3.11 does not understand '--coverage-for'.
```

Comment:
But this is ok.. because of the cuz_coverage enforcing a whole package build.

Reply:
You mean for me to add that line, saying that we don't need `--coverage-for` in Cabal < 3.11 because cuz_coverage enforces a whole pacakge build?